src/StatusQ/Components/StatusTagSelector.qml

```diff
@@ -378,6 +378,7 @@ Item {
         }
         contentItem: ListView {
             id: userListView
+            objectName: "tag-selector-user-list"
```

Comment:
So far we have use camelCase for object but we need a convention yet

Reply:
We always use camelCase in the QML code. We can keep snake_case for the python code only

Reply:
Ok! All modified!